### ISSUE TITLE
Link to the issue on the article page (Clean theme)

### ISSUE THREAD

**Issue by mattiker** [BirkbeckCTP/janeway#3328]
**Is your feature request related to a problem? Please describe.**
If you land on the article page directly via DOI, you may want to discover other articles from the issue it was published in (same or related topic). There is no direct link to the issue on the article page.

**Describe the solution you'd like**
A link to the issue of the article. Perhaps in form of breadcrumbs above the article title or as a link in the right column.

**Describe alternatives you've considered**
For a link in the right column you can use the "Additional Submission Fields" in the manager and generate a text area, but the link has to be set manually for every article in the metadata.

**Comment by ajrbyers** [comment]
@MatthiasULB is this for a specific journal/theme? I can see on Material & OLH there are links to the Issue, maybe is Clean specific?

**Comment by MatthiasULB** [comment]
@ajrbyers Yes, it is for the Clean theme. Sorry I should have been more specific.

**Comment by MatthiasULB** [comment]
Thanks for adding this issue on the 1.6 todo list. Since the other two templates have this function I thought that only minor tweaks would need to be made :) Until then, is there perhaps a simple interim solution without editing every single article metadata?

**Comment by MatthiasULB** [comment]
> Hey @MatthiasULB, I've added a tweak to your clean theme to display the issue in the short term. This will mirror the implementation.

@ajrbyers Awesome, thanks!!!

**Comment by MatthiasULB** [comment]
@ajrbyers is "Issues" translatable? The term does not change when I switch between English and German. For now, if that would be possible and would not cause a lot of effort, we would prefer the german term "Ausgabe" for the display.

**Comment by S-Haime** [comment]
Closed as completed
